lib/live-collection.js

```diff
@@ -255,6 +254,18 @@ module.exports = Backbone.Collection.extend({
     }, PATCH_TIMEOUT);
   },
 
+  resubscribeOnModelChange: function (model, prop){
+    var evtName = prop ? 'change:' + prop : 'change';
+    this.listenTo(model, evtName, this.resubscribe, this);
+  },
+
+  resubscribe: function (){
+    this.reset();
+    this.subscription.cancel();
+    this.subscription = null;
+    this.listen({ snapshot: true });
```

Comment:
Hmmm, I'm keen to get this change in as its relied upon in the webapp. Maybe file an issue and I can get onto it when the required feature have been merged in?